rebasehelper/application.py

```diff
@@ -440,8 +440,8 @@ def build_packages(self):
             build_dict['builds_nowait'] = self.conf.builds_nowait
             build_dict['build_tasks'] = self.conf.build_tasks
 
-            failed_before = False
-            while True:
+            files = {}
+            while self.conf.build_retries >= 0:
```

Comment:
I don't think we should decrease the configuration value. Instead I propose to save the value in a different variable and decrease it there or just store number of retries in a separate variable and check the value against the configuration value. The possible problem with current implementation is, that if rebase-helper retries builds of OLD RPMs, then it will not do the same number of retires for NEW RPMs. In a corner case the building of NEW RPMs will not start at all if the value is decreased to -1.